cli/index.js

```diff
@@ -40,37 +49,32 @@ const cli = meow(`
     --output-path     The location to output the response(default=stdout)
 `);
 
-const defaultUrl = 'https://operasoftware.github.io/pwa-list/';
-const url = cli.input[0] || defaultUrl;
+const url = cli.input[0] || 'https://pwa.rocks/';
+const outputMode = cli.flags.output || 'pretty';
 const outputPath = cli.flags.outputPath || 'stdout';
+const flags = cli.flags;
 
-if (semver.lt(process.version, '5.0.0')) {
-  console.error('Lighthouse requires node version 5.0 or newer');
-  process.exit(1);
-}
+const driver = new ChromeProtocol();
 
-lighthouse({
-  url: url,
-  flags: cli.flags
-}).then(results => {
-  const outputMode = cli.flags.output || 'pretty';
-  return Printer.write(results, outputMode, outputPath);
-})
-.then(status => {
-  if (outputPath !== 'stdout') {
-    log.info('printer', status);
-  }
-})
-.catch(err => {
-  if (err.code === 'ECONNREFUSED') {
-    console.error('Unable to connect to Chrome. Did you run ./launch-chrome.sh?');
-  } else {
-    console.error('Runtime error encountered:', err);
-    console.error(err.stack);
-  }
-  process.exit(1);
-});
+// kick off a lighthouse run
+lighthouse(driver, {url, flags})
+  .then(results => {
+    return Printer.write(results, outputMode, outputPath);
+  })
+  .then(status => {
+    outputPath !== 'stdout' && log.info('printer', status);
+  })
+  .catch(err => {
+    if (err.code === 'ECONNREFUSED') {
+      console.error('Unable to connect to Chrome. Did you run ./launch-chrome.sh?');
+    } else {
+      console.error('Runtime error encountered:', err);
+      console.error(err.stack);
+    }
+    process.exit(1);
+  });
 
+// set logging preferences
```

Comment:
logging level should probably be set above the `lighthouse()` call above. No logging in there currently, but if we ever add some, we'll want the logging level already set.